components/GeneralModal/GeneralModal.jsx

```diff
@@ -5,7 +5,7 @@ import { TouchableOpacity, View, ScrollView } from "react-native";
 export default function GeneralModal({ closeModal, children }) {
     return (
         <View className="flex flex-1 justify-center items-center bg-[#FFFFFF90]">
-            <View className={`w-10/12 h-2/5 border-2 rounded-xl  bg-white`}>
+            <View className={`w-10/12 h-fit pb-4 border-2 rounded-xl  bg-white`}>
```

Comment:
What does fit do?

Reply:
fit the box to the size of the content. then padding at the bottom to make it evenly spaced around the modal so the bottom line isn't basically touching the submit button

Reply:
not really relevant to my pr tbf just was annoying me

Reply:
Works for me :)